tests/pools/test_pool_rpc.py

```diff
@@ -400,7 +398,7 @@ async def test_create_multiple_pool_wallets(self, one_wallet_node_and_rpc, fee,
 
         # Test creation of many pool wallets. Use untrusted since that is the more complicated protocol, but don't
         # run this code more than once, since it's slow.
-        if fee == 0 and not trusted:
+        if not trusted:
```

Comment:
this change seems unrelated

Reply:
I see. so, technically, this is a separate, independent, test parameter.

Reply:
yeah good point, I guess i can parametrize that differently